src/digital_identity_service/share.qr.code.create.result.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
```

Comment:
What is that file about ??? This is related to the other PR ? Am I not meant to look at the retrieval of a session ?